### DISCUSSION TITLE
An additional black box is opened when running this program.

### DISCUSSION THREAD

**Discussion by minghamn** [hecrj/iced#1605] [Q&A] [answered]
When I use this to write a local application, running the exe file opens a black window screen.

![image](https://user-images.githubusercontent.com/95327699/207499025-f0f29c12-db48-42a1-aa83-6cf3da115aec.png)

**Comment by 13r0ck** [comment, marked as answer]
That's called the windows command prompt. It's really useful for debugging software. If your code has a print statement in it, you will see it there.

It can be disabled by adding `#![windows_subsystem = "windows"]` to your code [like this](https://github.com/13r0ck/bathtub/blob/main/src/main.rs#L2), but you probably want it there while you develop your application.

**Reply by minghamn** [reply]
I got it, thanks for the answer.
